src/components/MathSymbolsPicker/MathSymbolsPicker.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import selectors from 'selectors';
+
+import './MathSymbolsPicker.scss';
+
+const dataElement = 'colorPalette';
+
+class MathSymbolsPicker extends React.PureComponent {
```

Comment:
New components should use React function component rather than class components

Reply:
ok, I have converted this to a functional component.